extensions/arc/src/ui/dashboards/postgres/postgresParametersPage.ts

```diff
@@ -373,14 +373,14 @@ export class PostgresParametersPage extends DashboardPage {
 		if (!this.searchBox!.value) {
 			this.parametersTable.setFilter(undefined);
 		} else {
-			this.filterParameters(this.searchBox!.value);
+			this.filterParameters(this.searchBox!.value.toUpperCase());
```

Comment:
I'd suggest putting all the toUpperCase calls in the filterParameters function so it's guaranteed to be applied correctly even if that's called from somewhere else in the future.